docs/modules/customize/pages/cookies.adoc

```diff
@@ -0,0 +1,80 @@
+= Cookies
+
+With cookie consent we can explain better what cookies do we use, why we're using these cookies, and let the user choose which cookies they want to use.
+
+== Different cookie categories available in Decidim
+
+By default there are four (4) cookie categories: **essential**, **preferences**, **analytics** and **marketing**. To display embedded iframes properly user has to give consent to all categories.
+
+== Adding scripts which require cookie consent
+
+Recommended place for scripts is ```app/views/layouts/decidim/_head_extra.html.erb```.
+For example if you want to add script that requires marketing consent, you can add following code:
+
+```
+<script type="text/plain" data-consent="marketing">
+  console.log('marketing cookies accepted');
+</script>
+```
+
+**Note that script type should be "text/plain"!** Without that user's cookie consent setting doesn't affect.
+
+== Adding cookie category and cookie documentation
+
+You are able to modify cookie categories via consent_gategories configuration option.
+
+For example if you want to add "statistics" category you need to add following code to your app's initializer:
+
+```
+Decidim.configure do |config|
+  config.consent_categories = [
```

Comment:
Can you add this commented out at https://github.com/decidim/decidim/blob/develop/decidim-generators/lib/decidim/generators/app_templates/initializer.rb, so it's available in newly generated apps? Thanks